setup.py

```diff
@@ -47,7 +47,7 @@ def check_dependencies():
                             'clean'))):
```

Comment:
Also for the runtime, I would want to be able to do `pydeface.py --help` and see the complete options even without fsl installed. Right now it's the same either way:

```
$pydeface --help
FSL must be installed and FSLDIR environment variable must be defined
```